src/Tools/Source/RunTests/TestRunner.cs

```diff
@@ -156,8 +161,6 @@ private void Print(List<TestResult> testResults)
                 errorOutput = string.Format("Command: {0} {1}", _xunitConsolePath, builder.ToString())
                     + Environment.NewLine
                     + errorOutput;
-
-                Process.Start(resultsPath);
```

Comment:
@mmitche this should be done in the non-xml case because it essentially auto-opens the HTML file in the browser as the test fail.  Really easy to see the failure that way.